app/models/bot.rb

```diff
@@ -91,4 +95,22 @@ def self.generate_api_secret
       break token unless Bot.exists?(api_secret: token)
     end
   end
+
+  def validate_minimum_cover_image_size
+    if cover.path
+      image = MiniMagick::Image.open(cover.path)
+      unless image[:width] > 640 && image[:height] > 180
+        errors.add :cover, "should be 640x180px minimum!" 
+      end
+    end  
+  end
+
+  def validate_maximum_cover_image_size
```

Comment:
Can't send 1280x360px image. But 1279x359px...
This is a Dev cover problem too. I create the issue #223 to resolve it.